lib/dfe/analytics/event.rb

```diff
@@ -5,7 +5,14 @@
 module DfE
   module Analytics
     class Event
-      EVENT_TYPES = %w[web_request create_entity update_entity delete_entity import_entity].freeze
+      EVENT_TYPES = %w[
+        web_request
+        create_entity
+        update_entity
+        delete_entity
+        import_entity
+        analytics_initialise
```

Comment:
@asatwal maybe ignore this (!) but given we've used verb_noun form for the entity event types what would you think of using ```initialise_analytics``` for this new event type for consistency?

Reply:
+1!

Reply:
@stevenleggdfe yes `initialise_analytics` sounds better !